pkg/scheduler/framework/interface.go

```diff
@@ -307,10 +307,14 @@ type QueueSortPlugin interface {
 }
 
 // EnqueueExtensions is an optional interface that plugins can implement to efficiently
-// move unschedulable Pods in internal scheduling queues.
+// move unschedulable Pods in internal scheduling queues. Plugins
+// who will fail pod scheduling should implement this interface.
+// With registered events, scheduler can build dynamically event handlers.
```

Comment:
It's described in L316~L317, so let's remove it.

Reply:
yep, it's duplicated👍